src/requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+-r requirements.txt
+
+pytest
+pytest-asyncio                        
+pytest-cov                            
+pytest-flakefinder                     
+pytest-forked                          
```

Comment:
`pytest-forked` is a dependency of `pytest-xdist` and therefore does not not need to be listed explicitly in the requirements.

Reply:
fixed